projects/core/src/button/button.element.ts

```diff
@@ -92,25 +107,26 @@ export class CdsButton extends CdsBaseButton {
   }
 
   update(props: PropertyValues<this>) {
-    if (props.has('loadingState')) {
-      if (this.isDefaultLoadingState(props.get('loadingState') as string)) {
-        // track prior disabled state to set prior value after button is re-enabled from a loading state
-        this._disabled = this.disabled;
-      }
-
-      if (props.get('loadingState') !== undefined) {
-        if (this.isDefaultLoadingState(this.loadingState)) {
-          this.enableButton();
-        } else {
-          this.disableButton();
-        }
+    if (props.has('loadingState') && props.get('loadingState') !== undefined) {
+      if (this.isDefaultLoadingState(this.loadingState)) {
```

Comment:
Why was this changed from `props.get('loadingState')` to accessing the property directly with `this.loadingState`?